### PR TITLE
Add "matched" parameter to expect callback

### PR DESCRIPTION
**Pull request by stacywile**

### CODE REVIEW

**Comment by stacywile** [comment]
I don't understand why the travis CI build failed. Anybody want to help?

File: lib/spectcl.js [hunk 1]
```diff
@@ -258,6 +260,7 @@ Spectcl.prototype.expect = function(expArr, cb){
          * @returns {String|RegExp|Number} The Expectation that was found in the cache or null if no match was found.
          */
       , matchCache = function(){
+          debug('[expect] self.cache = %s',self.cache);
```
Comment (gcochard) [inline lib/spectcl.js]: Drop the semicolon here

File: lib/spectcl.js [hunk 2]
```diff
@@ -271,12 +274,13 @@ Spectcl.prototype.expect = function(expArr, cb){
                   debug('[expect] [%d] match FULL_BUFFER %s', self.child.pid, matched)
                   return expectations[i]
               }
-              if(exp instanceof RegExp){
+              if(exp instanceof RegExp){
```
Comment (gcochard) [inline lib/spectcl.js]: Get rid of the extra whitespace at the end of this line, after the brace

**Comment by stacywile** [comment]
All checks have passed!

**Comment by stacywile** [comment]
Can someone please approve this

**Comment by ryanbmilbourne** [comment]
Thanks for taking the time to modify per our request in your earlier PR.
:+1: :rocket: :ship: :shipit:

**Pull request merged**